File: crates/siloscan-core/tests/profiles-corpus/tree/typescript/nesting-depth-under.ts
export function walkUnder(values: number[]): number {
  let total = 0;
  for (const value of values) {
    for (const value of values) {
      for (const value of values) {
        for (const value of values) {
          for (const value of values) {
            total += value;
          }
        }
      }
    }
  }
  return total;
}

File: crates/siloscan-core/tests/profiles-corpus/tree/go/nesting-depth-under.go
package p

func WalkUnder(values []int) int {
	total := 0
	for _, value := range values {
		for _, value := range values {
			for _, value := range values {
				for _, value := range values {
					for _, value := range values {
						total += value
					}
				}
			}
		}
	}
	return total
}
